### PR TITLE
lambo, lambo-new: add page

### PR DESCRIPTION
**Pull request by owenvoke**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by bl-ue** [comment]
Is there a reason why you sometimes use `=` in flags but sometimes not?
Like this:
```
lambo new --path={{path/to/directory}} {{name}}

lambo new --auth {{name}}

**Comment by owenvoke** [comment]
@bl-ue, because the ones without an equals sign don't have a value (they are a boolean value). Whereas the ones with `=` need a value.

**Comment by bl-ue** [comment]
Seems like an obvious one now 😄

**Review by bl-ue — APPROVED** [review]

**Review by navarroaxel — COMMENTED** [review]
I think this should be move to `lambo-new.md` and create a `lambo.md` for `--help`, `edit-config`, `make-after`, `edit-after` and the simplest `lambo new` subcommands.

File: pages/common/lambo.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# lambo
+
+> A super-powered `laravel new` for Laravel and Valet.
+> More information: <https://github.com/tighten/lambo>.
+
+- Create a new Laravel application:
+
+`lambo new {{name}}`
```
Comment (sbrl) [inline pages/common/lambo.md:8]: Maybe this would aid clarity? Same goes for the rest of the examples.
```suggestion
`lambo new {{app_name}}`
```

Comment (owenvoke) [reply pages/common/lambo.md:8]: Yeah, I did think about that. I was just being consistent with other pages.

Comment (sbrl) [reply pages/common/lambo.md:8]: Ah, ok. Still, I think this adds enough clarity to consider breaking consistency.

I guess it doesn't really matter then.

Comment (owenvoke) [reply pages/common/lambo.md:8]: I'll update this shortly. Cheers.

**Comment by navarroaxel** [comment]
Hey @owenvoke, do you think this is not useful? https://github.com/tldr-pages/tldr/pull/5139#pullrequestreview-568185967

**Comment by owenvoke** [comment]
Apologies @navarroaxel, I missed that comment. I'm not entirely sure that this would be necessary. 🤔

**Comment by bl-ue** [comment]
I'm inclined to agree with @navarroaxel, because what about the other `lambo` subcommands like `edit-config`, `make-after `, and `edit-after`?  Just because they may not be very useful/used often doesn't mean we shouldn't document them, you know.

**Comment by owenvoke** [comment]
Fair enough, I was just showing the most used commands, for reference `make-after` no longer exists as it's irrelevant, and there isn't a `--help` flag (it's displayed when no command or directory is specified). But I've added now split it into another page with the other commands.

**Review by bl-ue — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Comment by bl-ue** [comment]
Oh cool!  I didn't notice `lambo` _in addition to_ `lambo-new` :smile:

Thank you @owenvoke!

**Pull request merged**